docs/frontend.md

```diff
@@ -167,9 +167,9 @@ To serve SSR markup to the client, you need to register the **live-require-handl
 apiHandlers: [
```

Comment:
Can you make the following changes above?

Line 115:
`To use Server Side Rendering (SSR) in your application, you need to implement one or several handlers that pre-render the client application's markup on the server.`

Can you update this line as follows?

`To use Server-Side Rendering (SSR) in your application, you need to implement one or more handlers that pre-render the client application's markup on the server.`

Line 135:
`To enable server side rendering, specify an array of server side rendering scripts that target different environments in the `clientEntries` configuration section:`

Please change "server side rendering" to "server-side rendering", and "server side rendering scripts" to "server-side rendering scripts".

Reply:
Done